src/code/audio_synthesis_data.c

```diff
@@ -0,0 +1,691 @@
+    0.327776f, 0.316066f, 0.304308f, 0.292503f, 0.280653f, 0.268761f, 0.256827f, 0.244854f, 0.232844f, 0.220798f,
+    0.208718f, 0.196606f, 0.184465f, 0.172295f, 0.160098f, 0.147877f, 0.135634f, 0.12337f,  0.111087f, 0.098786f,
+    0.086471f, 0.074143f, 0.061803f, 0.049454f, 0.037097f, 0.024734f, 0.012368f, 0.0f,
+};
+
+s16 sLowPassFilterData[] = {
+    /* 0x0 */ 0,     0,     0,    32767, 0,    0,     0,     0, // Identity filter (delta function)
+    /* 0x1 */ 3854,  4188,  4398, 4469,  4398, 4188,  3854,  3416,
+    /* 0x2 */ 3415,  4314,  4915, 5126,  4915, 4314,  3415,  2351,
+    /* 0x3 */ 2636,  4433,  5762, 6252,  5762, 4433,  2636,  849,
+    /* 0x4 */ 1334,  4196,  6646, 7609,  6646, 4196,  1334,  -802,
+    /* 0x5 */ -265,  3421,  7292, 8944,  7292, 3421,  -265,  -1863,
+    /* 0x6 */ -1558, 2065,  7146, 9546,  7146, 2065,  -1558, -1682,
+    /* 0x7 */ -2353, 726,   7441, 11028, 7441, 726,   -2353, -697,
+    /* 0x8 */ -2252, -693,  7121, 11962, 7121, -693,  -2252, 668,
+    /* 0x9 */ -1373, -1819, 6299, 12298, 6299, -1819, -1373, 1484,
+    /* 0xA */ -213,  -2740, 5843, 13680, 5843, -2740, -213,  1494,
+    /* 0xB */ 980,   -3081, 4883, 14286, 4883, -3081, 980,   590,
+    /* 0xC */ 1769,  -2973, 3866, 14981, 3866, -2973, 1769,  -568,
+    /* 0xD */ 2023,  -2554, 2911, 16397, 2911, -2554, 2023,  -1391,
+    /* 0xE */ 1766,  -1918, 2016, 19800, 2016, -1918, 1766,  -1564,
+    /* 0xF */ 841,   -853,  863,  26829, 863,  -853,  841,   -820,
+};
+
+s16 sHighPassFilterData[] = {
+    /* 0x0 */ -289,  -291,  -289,  30736, -289,  -291,  -289,  -290,
+    /* 0x1 */ -464,  -467,  -467,  29506, -467,  -467,  -464,  -463,
+    /* 0x2 */ -662,  -670,  -672,  28101, -672,  -670,  -662,  -656,
+    /* 0x3 */ -839,  -855,  -861,  26830, -861,  -855,  -839,  -822,
+    /* 0x4 */ -996,  -1024, -1038, 25685, -1038, -1024, -996,  -963,
+    /* 0x5 */ -1184, -1236, -1266, 24272, -1266, -1236, -1184, -1118,
+    /* 0x6 */ -1357, -1450, -1506, 22900, -1506, -1450, -1357, -1238,
+    /* 0x7 */ -1514, -1680, -1784, 21498, -1784, -1680, -1514, -1307,
+    /* 0x8 */ -1613, -1877, -2048, 20390, -2048, -1877, -1613, -1298,
+    /* 0x9 */ -1657, -2185, -2559, 18869, -2559, -2185, -1657, -1093,
+    /* 0xA */ -1524, -2395, -3078, 18030, -3078, -2395, -1524, -739,
+    /* 0xB */ -1253, -2504, -3621, 17642, -3621, -2504, -1253, -367,
+    /* 0xC */ -525,  -2367, -4732, 17517, -4732, -2367, -525,  0,
+    /* 0xD */ -34,   -1762, -5706, 17503, -5706, -1762, -34,   -258,
+    /* 0xE */ -772,  -3,    -6985, 17240, -6985, -3,    -772,  -3,
+};
+
+s16 D_80130418[] = {
+    0x0,    0x18F8, 0x30FB, 0x471C, 0x5A81, 0x6A6C, 0x7640, 0x7D89, 0x7FFF, 0x7D89, 0x7640,
+    0x6A6C, 0x5A81, 0x471C, 0x30FB, 0x18F8, 0x0,    0xF375, 0xE708, 0xDAD9, 0xCF05, 0xC3AA,
+    0xB8E4, 0xAECD, 0xA57F, 0x9D0F, 0x9594, 0x8F1F, 0x89C0, 0x8584, 0x8277, 0x809F, 0x8001,
+    0x809F, 0x8277, 0x8584, 0x89C0, 0x8F1F, 0x9594, 0x9D0F, 0xA57F, 0x9D0F, 0x9594, 0x8F1F,
+};
+
+u8 D_80130470[] = {
```

Comment:
Implementing this comment and the next took some steps but it looks like it's in a much more accurate state now. `D_80130520` wasn't being extracted before but now it is (it was part of an f32 previously) and the shift still matches. A separate file `audio_data.c` was compiled instead of included, and moving the data afterwards directly into Audio_synthesis allowed this to be the end of file. Removing the 0 pads still matches in that case.